### ISSUE TITLE
Feature request: Window Icon before window title

### ISSUE THREAD

**Issue by Rajaniraiyn** [nextapps-de/winbox#54]
Hi!
   I really loved your project I have planned to integrate this within [my project](https://github.com/Rajaniraiyn/windows11). We have implemented a similar window, but it is not much feature rich and extensible like yours.

Please add an API to add icon before title. that will be make this more good.

**Comment by mossaiby** [comment]
Hi @Rajaniraiyn I think this is possible using CSS if you use flexbox.

**Comment by Rajaniraiyn** [comment]
Yes we can do so. but if it is on this project natively then it will be very helpful for others too.
